packages/admin-panel/src/VizBuilderApp/components/DataLibrary/component/AggregateSelectedOptionWithJsonEditor.js

```diff
@@ -95,15 +95,15 @@ export const SelectedOptionWithJsonEditor = ({
       </FlexBetweenPanel>
       {isExpanded && (
         <JsonEditorPanel
-          onMouseOver={() => setEdittingOption(option.code)}
+          onMouseOver={() => setEdittingOption(option.id)}
           onMouseLeave={() => setEdittingOption(null)}
```

Comment:
I took me a really long time to work out what this `setEdittingOption` property was all about. It's just being used to control if the option is draggable or not? If that's the case can we rename it and have it behave a bit more clearly:
```suggestion
          onMouseOver={() => setIsDragDisabled(true)}
          onMouseLeave={() => setIsDragDisabled(false)}
```

Reply:
Yeah it makes it clearer. Thanks!